core/web/auth/gql.go

```diff
@@ -53,6 +56,9 @@ func SetGQLAuthenticatedSession(ctx context.Context, user clsessions.User, sessi
 // GetGQLAuthenticatedSession extracts the authentication session from a context.
 func GetGQLAuthenticatedSession(ctx context.Context) (*GQLSession, bool) {
 	obj := ctx.Value(sessionUserKey{})
+	if obj == nil {
+		return nil, false
+	}
```

Comment:
test on this case?

Reply:
Would be covered here: `core/web/auth/gql_test.go L31`, just updated to match `GetAuthenticatedUser`